leaflet-providers.js

```diff
@@ -175,7 +175,7 @@
 		},
 		MapBox: {
 			url: function (id) {
-				return 'http://{s}.tiles.mapbox.com/v3/' + id + '/{z}/{x}/{y}.png';
+				return '//{s}.tiles.mapbox.com/v3/' + id + '/{z}/{x}/{y}.png';
```

Comment:
this is typically a good thing but beware that some users will be on `file:///` urls and this as a bug (despite file urls being a bad idea)